frontend/src/utils/entity.ts

```diff
@@ -17,6 +22,20 @@ export const videoFromRelatedEntity = (entity: RelatedEntityObject): Video => {
   };
 };
 
+export const videoWithCriteriaFromRecommendation = (
+  entity: Recommendation
+): VideoSerializerWithCriteria => {
+  const video = videoFromRelatedEntity(entity);
+
+  return {
+    ...video,
+    criteria_scores: entity.criteria_scores,
+    tournesol_score: entity.tournesol_score || null,
```

Comment:
0 could be transformed into null here.
```suggestion
    tournesol_score: entity.tournesol_score ?? null,
```